sample/android-sdk-sample/app/src/main/res/layout/listview_header.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout android:layout_width="match_parent"
```

Comment:
Use `merge` instead for avoid deeper layout.

Reply:
How I can solve this issue:
https://stackoverflow.com/questions/13348455/can-you-use-the-merge-tag-with-fragments